pages/linux/zbarcam.md

```diff
@@ -0,0 +1,20 @@
+# zbarcam
+
+> Scans and decodes barcodes from a video device.
```

Comment:
I think most people aren't gonna think of QR codes when they read that, so I'd prefer to clarify a bit

```suggestion
> Scans and decodes barcodes (and QR codes) from a video device.
```

Reply:
I went to check that qr codes are called barcodes before committing, but I agree that most don't make the connection immediately.